rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -2367,6 +2232,69 @@ private String normalizeFilePath(String filePath, String fileName) {
         return filePath;
     }
 
+    @Override
+    public JobIdData reSubmit(String sessionId, String jobId, PathSegment pathSegment) throws JobCreationRestException,
+            NotConnectedRestException, PermissionRestException, SubmissionClosedRestException, IOException {
+        Scheduler scheduler = checkAccess(sessionId, "reSubmit");
+
+        File tmpJobFile = File.createTempFile("job", "d");
+
+        JobId newJobId;
+
+        try {
+            final String jobContent = scheduler.getJobContent(JobIdImpl.makeJobId(jobId));
+            FileUtils.write(tmpJobFile, jobContent, Charset.forName("UTF-8"));
+        } catch (NotConnectedException e) {
+            e.printStackTrace();
+        } catch (UnknownJobException e) {
+            e.printStackTrace();
+        } catch (PermissionException e) {
+            e.printStackTrace();
+        } catch (SubmissionClosedException e) {
+            e.printStackTrace();
+        } catch (JobCreationException e) {
+            e.printStackTrace();
+        }
+        // Get the job submission variables
+        Map<String, String> jobVariables = workflowVariablesTransformer.getWorkflowVariablesFromPathSegment(pathSegment);
+
+        // Get the job submission generic infos
+
+        WorkflowSubmitter workflowSubmitter = new WorkflowSubmitter(scheduler);
+        newJobId = workflowSubmitter.submit(tmpJobFile, jobVariables, Collections.emptyMap());
+        return mapper.map(newJobId, JobIdData.class);
+    }
+
+    @Override
+    public JobIdData reSubmit(String sessionId, String jobId) throws JobCreationRestException,
+            NotConnectedRestException, PermissionRestException, SubmissionClosedRestException, IOException {
+        Scheduler scheduler = checkAccess(sessionId, "reSubmit");
+
+        File tmpJobFile = File.createTempFile("job", "d");
+
+        JobId newJobId;
+
+        try {
+            final String jobContent = scheduler.getJobContent(JobIdImpl.makeJobId(jobId));
+            FileUtils.write(tmpJobFile, jobContent, Charset.forName("UTF-8"));
+        } catch (NotConnectedException e) {
+            e.printStackTrace();
+        } catch (UnknownJobException e) {
+            e.printStackTrace();
+        } catch (PermissionException e) {
+            e.printStackTrace();
+        } catch (SubmissionClosedException e) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Combine this catch with the one at line 2280, which has the same body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2147)